python/servo/testing_commands.py

```diff
@@ -351,7 +351,7 @@ def test_wpt_failure(self):
         ], env=self.build_env())
 
     @Command('test-wpt',
-             description='Run the web platform tests',
+             description='Run the web platform tests that the regular test suite',
```

Comment:
Personally speaking, It feel a bit redundant for me, I would write something like "Run the regular web platform test suite", or just leave it as it is originally.